### ISSUE TITLE
Redis storage

### ISSUE THREAD

**Issue by piotrkowalczuk** [piotrkowalczuk/mnemosyne#11]
Mnemosyne needs alternative way to store sessions. Redis storage is one of missing ones.

**Comment by klebervirgilio** [comment]
Hey @piotrkowalczuk! is the idea to implement the following actions for redis storage?
 Get
 List
 Exists
 Create
 Abandon
 SetData
 Delete
 Setup
 TearDown

**Comment by piotrkowalczuk** [comment]
Yes you get it right! Such storage would have to pass already existing set of test scenarios.
